src/MentoringAndCoaching/schema-build-helpers.ts

```diff
@@ -0,0 +1,141 @@
+import moment from "moment";
+import { v4 as uuid } from "uuid";
+import { MentoringContext } from "./schema";
+
+export const buildContext = async (input: any = {}): Promise<any> => {
```

Comment:
Was there any reason to make this function **async** ? If not any speicific, can we remove this.

Reply:
done